inst/include/population_dynamics/population/population.hpp

```diff
@@ -457,7 +457,7 @@ struct Population : public fims_model_object::FIMSObject<Type> {
                  this->weight_at_age[age];
       } else {
         POPULATION_LOG << "fleet " << fleet_ << " is a survey" << std::endl;
-        index_ = this->fleets[fleet_]->q *
+        index_ = this->fleets[fleet_]->q[0] *
```

Comment:
q is being hard coded to a single value here which doesn't allow this parameter to be time varying. We need code to detect if this parameter is of size one in which case it is treated as a scalar or if the size is nyears in which it gets indexed by year.

Reply:
This has been fixed to q.get_force_scalar(year)